### PR TITLE
adding hometown file for lhalley

### PR DESCRIPTION
**Pull request by lhalley**
@yummytree  what kind of restaurants do you like so I can add them to the hometown file?

### CODE REVIEW

File: lhalley-schaumburg.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Schaumburg, IL
+
+### Restaurants
+- Texas do Brasil
+### Attractions
```
Comment (yummytree) [inline lhalley-schaumburg.md]: I like :hamburger: from Portillos, and some :kr:  food.

**Review by yummytree — CHANGES_REQUESTED** [review]
added some restaurants, ready to :ship:

File: lhalley-schaumburg.md (hunk 1 above)
Comment (lhalley) [reply lhalley-schaumburg.md]: Added some restaurants, let me know if it's ok to ship

**Review by yummytree — APPROVED** [review]
this looks good!

**Pull request merged**